Austin_Marga_HW7_2.ipynb

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
<p>That is not the most pronounced periodic structure. Look eg at Luke's example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_2DFFT/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>